### DISCUSSION TITLE
My Copliot Stopped Giving Inline suggestions.

### DISCUSSION THREAD

**Discussion by spragginsdesigns** [community/community#10598] [Copilot Conversations] [answered]
I was using GitHub Copilot for the last month, and it seemed to work fine. I still get some emmet and other popups for snippets but nothing inline. 
I usually have to enter Crtl + Enter to obtain suggestions; otherwise, it wouldn't give me inline suggestions. What am I doing wrong? 

I have uninstalled extensions I do not need. I have reinstalled it. The 10/10 suggestions works, but no inline only emmet or something else, even on basic HTML, CSS and JS files. 

Here is a screenshot of a basic HTML file, and I can only get suggestions, not inline ones. 
![image](https://user-images.githubusercontent.com/43624199/151246111-a4c3aebb-9a29-4fb3-a8a9-ea6a75800cc7.png)

After I choose a suggestion and try to make a navbar, it just shows snippet suggestions and nothing inline. I have no idea why and was wondering if anyone else knew? Thank you. 
![image](https://user-images.githubusercontent.com/43624199/151246358-04ab19b0-9d6b-4c89-b324-be794535a565.png)

**Comment by spragginsdesigns** [comment, marked as answer]
Never mind, my "Inline Suggestions" were turned off in settings. What a facepalm. I couldn't believe it had stopped working. 
Is there anything else in settings that should be turned on or off? Or anything to put in the JSON settings? 

![image](https://user-images.githubusercontent.com/43624199/151247346-6d459a51-2e3d-4bc9-9c8f-b98f4e9f0135.png)

**Comment by franzitobambino** [comment]
Thanks for this (face in palm)
